brave/browser-laptop
# Delaying ref headers fetch until after startup

Fixes #14615 

## Submitter Checklist:


## Test Plan:

1. Create empty profile folder
2. Create a file called `promoCode` in the profile folder with a test promo code, such as ~`kam253`~
3. Run Brave with `LEDGER_VERBOSE=true and LEDGER_ENVIRONMENT=staging`
4. Confirm promotion code is read after 50 - 70 seconds (you should see call in the terminal)
5. Make sure that `promoCode` file is deleted when using real promo code

## Reviewer Checklist:


Tests
